src/modm/driver/temperature/max31865_impl.hpp

```diff
@@ -0,0 +1,115 @@
+// coding: utf-8
+// ----------------------------------------------------------------------------
+/*
+ * Copyright (c) 2023, Henrik Hose
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_MAX31865_HPP
+#error "Don't include this file directly, use 'max31865.hpp' instead!"
+#endif
+
+namespace modm
+{
+
+template<typename SpiMaster, typename Cs, Pt pt>
+Max31865<SpiMaster, Cs, pt>::Max31865(max31865<pt>::Data &data) : data(data)
+{
+	this->attachConfigurationHandler([]() { SpiMaster::setDataMode(SpiMaster::DataMode::Mode3); });
+	Cs::setOutput(modm::Gpio::High);
+}
+
+template<typename SpiMaster, typename Cs, Pt pt>
+modm::ResumableResult<void>
+Max31865<SpiMaster, Cs, pt>::initialize()
+{
+
+	config = typename max31865<pt>::Config();
+	config.set(max31865<pt>::Config::FaultStatusClear);
+	config.set(typename max31865<pt>::Rejection_t(max31865<pt>::Rejection::Rejection50Hz));
+	RF_CALL_BLOCKING(writeSingleRegister(max31865<pt>::Register::WriteConfiguration, config.value));
+	config = typename max31865<pt>::Config_t(
+		RF_CALL_BLOCKING(readSingleRegister(max31865<pt>::Register::ReadConfiguration)));
+}
```

Comment:
You need to use `RE_BEGIN();`/`RF_END();` in a resumable function, and then replace `RF_CALL_BLOCKING` by `RF_CALL`.

```suggestion
template<typename SpiMaster, typename Cs, Pt pt>
modm::ResumableResult<void>
Max31865<SpiMaster, Cs, pt>::initialize()
{
	RF_BEGIN();
	config = Config();
	config.set(Config::FaultStatusClear);
	config.set(Rejection_t(Rejection::Rejection50Hz));
	RF_CALL(writeSingleRegister(Register::WriteConfiguration, config.value));
	config = Config_t(RF_CALL(readSingleRegister(Register::ReadConfiguration)));
	RF_END();
}
```